tests/unit/test_stores.py

```diff
@@ -79,13 +79,28 @@ async def test_expires(store: Store, frozen_datetime: Coordinates) -> None:
 @pytest.mark.flaky(reruns=5)
 @pytest.mark.parametrize("renew_for", [10, timedelta(seconds=10)])
 async def test_get_and_renew(store: Store, renew_for: int | timedelta, frozen_datetime: Coordinates) -> None:
+    if isinstance(store, RedisStore):
+        pytest.skip()
+
     await store.set("foo", b"bar", expires_in=1)
     await store.get("foo", renew_for=renew_for)
 
     frozen_datetime.shift(2)
 
-    if isinstance(store, RedisStore):
-        await asyncio.sleep(1.1)
+    stored_value = await store.get("foo")
+
+    assert stored_value is not None
+
+
+@pytest.mark.flaky(reruns=5)
+@pytest.mark.parametrize("renew_for", [10, timedelta(seconds=10)])
+async def test_get_and_renew_redis(store: RedisStore, renew_for: int | timedelta) -> None:
```

Comment:
Probably an oversight, the `store` fixture provides all stores, but the name suggests this test is only for `RedisStore`s

Reply:
Fixed!